model1.patch

```diff
@@ -0,0 +1,225 @@
+diff --git a/compile.sh b/compile.sh
```

Comment:
I have made changes as per your suggestion. Please have a look at it in updated code.